src/packages/templating/stylesheets/components/stylesheet-rule-input/stylesheet-rule-input.element.ts

```diff
@@ -32,18 +32,24 @@ export class UmbStylesheetRuleInputElement extends FormControlMixin(UmbLitElemen
 	}
 
 	#appendRule = async () => {
```

Comment:
This doesn't need to be async anymore:

```suggestion
	#appendRule = () => {
```